ignition/service/logging.py

```diff
@@ -58,10 +70,42 @@ def format(self, record):
         result = self._obfuscate_sensitive_data(result)
         return result
 
+    def _mask_sensitive_data(self, suffix, prefix, message, rep=None):
+        regex = re.compile('{0}(.*?){1}'.format(prefix, suffix), flags=re.DOTALL | re.IGNORECASE)
+        abc = re.findall(regex, message)
```

Comment:
Use proper name instead of abc as variable name. Only the first keyword is handled, what if there there are more than 1 keywords found.

Reply:
Added loop to handle multiple occurrences